checker/tests/nullness/RequiresNonNullTest.java

```diff
@@ -143,4 +143,10 @@ void notHiddenTest() {
     void test() {
         o = null;
     }
+
+    @RequiresNonNull("thisShouldIssue1Error")
+    // Test case for Issue 1051
+    // https://github.com/typetools/checker-framework/issues/1051
+    //:: error: (flowexpr.parse.error)
```

Comment:
The error would be much better on the `@RequiresNonNull`. A method might have multiple annotations that use flow expressions and a single error on the method isn't very useful.
I couldn't find an issue to update this. Should I create one?